dev.Dockerfile

```diff
@@ -37,7 +37,15 @@ COPY docker/dev/Settings-dev.toml /app/Settings.toml
 COPY docker/dev/service.sh /etc/service/geoengine/run
 RUN chmod +x /etc/service/geoengine/run \
     && \
-    adduser --disabled-password --gecos "" geoengine
+    adduser --disabled-password --gecos "" geoengine 
+
+# Create upload folder and chown it to the geoengine user
+RUN mkdir -p /app/upload \
+    && \
+    chown -R geoengine:geoengine /app/upload \
+    && \
+    chown -R geoengine:geoengine /app/operators/test-data/vector/data
+# the last line is a work-around to allow the ogr-source to read gpkg files from the test-data folder. The gpkg driver needs RW rights.
```

Comment:
have you tried opening the data set as read only? Or, alternatively, is it possible to modify the gpkg s.t. it doesn't need write access?

Reply:
i don' think that works